python/cudf/cudf/tests/test_multiindex.py

```diff
@@ -709,6 +709,14 @@ def test_multiindex_groupby_reset_index():
     assert_eq(pdg.reset_index(), gdg.reset_index())
 
 
+def test_multicolumn_reset_index():
```

Comment:
Let's add a situation where we have a multi column grouping key so that we get a MultiIndex as well.

Reply:
done below